### PR TITLE
Fixing top bar

### PR DESCRIPTION
**Pull request by christian140398**
Closes #{ISSUE_ID}

### Changes/Additions:

- Fixed checkbox

### CODE REVIEW

**Review by DenFlyvendeGed — APPROVED** [review]
LGTM

File: src/lib/components/topBar/TopBar.svelte [hunk 1]
```diff
@@ -224,7 +224,10 @@
 			icon={Error}
 			name="About"
 			on:click={() => {
-				console.log("About");
+				var answer = window.confirm("This will open Ecdars webpage.");
+				if (answer) {
+					window.open("https://www.ecdar.net/");
```
Comment (TSKsmiley) [inline src/lib/components/topBar/TopBar.svelte:229]: this does not seem to work in tauri (have only tested on mac)

File: package.json [hunk 2]
```diff
@@ -38,6 +38,7 @@
 		"@floating-ui/dom": "^1.5.3",
 		"@tauri-apps/api": "^1.5.1",
 		"svelte-google-materialdesign-icons": "^0.8.2",
+		"yarn": "^1.22.21",
```
Comment (atjn) [inline package.json:41]: This is telling yarn to bundle the yarn source code with the web app.

File: src/lib/components/topBar/TopBar.svelte [hunk 3]
```diff
@@ -224,16 +226,69 @@
...
+				aboutVisible = true;
 			}}
 		/>
 	</TopBarButton>
 </div>
 
+{#if aboutVisible}
+	<div class="pageblocker">
+		<div class="aboutBox">
+			<div style="position: absolute; left: 2%;">
```
Comment (atjn) [inline src/lib/components/topBar/TopBar.svelte:238]: Please use the standardized modal that is being merged here: https://github.com/ECDAR-AAU-SW-P5/Ecdar-GUI-Web/pull/111/files#diff-4fa9b2f0e22c217008f008d25d07f473d343030aca709b90d8ab2bfb57d24770

Any popup should use this modal to ensure consistency in the UI.

**Review by atjn — CHANGES_REQUESTED** [review]

**Review by KamyaPA — APPROVED** [review]
LGTM

**Review by atjn — APPROVED** [review]
LGTM

**Pull request merged**